apps/src/code-studio/pd/workshop_dashboard/reports/foorm/results.jsx

```diff
@@ -55,9 +58,26 @@ export default class Results extends React.Component {
             title="Workshop Rollups"
           >
             <SurveyRollupTableFoorm
-              workshopRollups={this.props.workshopRollups}
+              workshopRollups={this.props.workshopRollups[GENERAL]}
               questions={this.props.questions}
               courseName={this.props.courseName}
+              isPerFacilitator={false}
+              facilitators={this.props.facilitators}
+            />
+          </Tab>
+        )}
+        {this.props.workshopRollups && this.props.workshopRollups[FACILITATOR] && (
+          <Tab
+            eventKey={this.props.workshopTabs.length + 2}
```

Comment:
Should we explain what the `+ 2` (and maybe `+ 1` below) are for?  Or extract them in variables?

Reply:
adding a comment, since this number is calculated based off the number of tabs previously created it would be messy to make a variable